lib/templates/android_screen_base.tt

```diff
@@ -183,4 +183,25 @@ class AndroidScreenBase < Calabash::ABase
     # Clicking in the Done button
     touch "* id:'button1'"
   end
+
+
+
+  # @param [String] id The view's id
+  # @param [String] The complete view query (optional)
+  # @return [Array] return an array[String] associated with the views
+  def text_array_from(id, query_text = nil)
```

Comment:
All other methods use only `query` as a parameter.
I think that would be nice to follow these pattern.